src/main/scala/cromwell/util/ConfigUtil.scala

```diff
@@ -26,14 +26,20 @@ object ConfigUtil {
 
     def keys = config.entrySet().toSet map { v: java.util.Map.Entry[String, ConfigValue] => v.getKey }
 
-    def getStringOption(key: String): Option[String] = {
```

Comment:
Not sure if this is the appropriate time, but this looks like a candidate for Lenthallization.